dali/ft/filecopy.cpp

```diff
@@ -2033,6 +2033,8 @@ void FileSprayer::gatherFileSizes(FilePartInfoArray & fileSizeQueue, bool errorI
             if (alldone)
                 break;
         }
+        for (idx = 0; idx < numThreads; idx++)
+            threads.item(idx).join();
```

Comment:
This looks like an unrelated fix (and change below between 2571->2576).
Move into a different PR?

Reply:
Yes.  I will move.